### PR TITLE
Added a remove slot function to canvas

### PR DESCRIPTION
**Pull request by valeriegc**

### CODE REVIEW

File: src/routes/Canvas.svelte [hunk 1]
```diff
@@ -5,6 +5,11 @@
 	let resizing = true;
 	let currentIndex: number;
 
+	function removeArea(i:number){
+     $mapObjects.splice(i,1)
+	 $mapObjects = $mapObjects
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]: Missing indentation

File: src/routes/Canvas.svelte [hunk 2]
```diff
@@ -44,7 +49,7 @@
 	<div class="completeWrap">
 		<div class="imageCanvasWrap" on:mousedown={logMouseDown}>
 			<img src={URL.createObjectURL($files[0])} />
-			{#each $mapObjects as coordinate, i}
+			{#each $mapObjects as _, i}
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte:52]: Nice!

File: src/routes/Canvas.svelte [hunk 3]
```diff
@@ -66,6 +71,12 @@
 						on:mousedown|stopPropagation={(e) => resizeStart(e, i)}
 						on:mouseup={resizeEnd}
 					/>
+					<!-- svelte-ignore a11y-click-events-have-key-events -->
+					<div 
+						class="remove"
+						style="position:absolute;top:-14px; right: -14px; overflow:visible"
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]: If you are adding a class, maybe style everything on the class

File: src/routes/Canvas.svelte [hunk 4]
```diff
@@ -107,4 +118,12 @@
 		border: solid 3px darkblue;
 		background-color: whitesmoke;
 	}
+	.remove{
+		height:12px;
+		width: 12px;
+		border-radius: 100%;
+		background-color:transparent;
+		color:darkred;
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]:
```suggestion
		background-color: transparent;
		color: darkred;
```

File: src/routes/Canvas.svelte [hunk 5]
```diff
@@ -107,4 +118,12 @@
 		border: solid 3px darkblue;
 		background-color: whitesmoke;
 	}
+	.remove{
+		height:12px;
+		width: 12px;
+		border-radius: 100%;
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]: Is this needed?

**Review by AlexanderKolberg — CHANGES_REQUESTED** [review]

File: src/routes/Canvas.svelte (hunk 1 above)
Comment (valeriegc) [reply src/routes/Canvas.svelte]: fixed

File: src/routes/Canvas.svelte (hunk 3 above)
Comment (valeriegc) [reply src/routes/Canvas.svelte]: Resolved

File: src/routes/Canvas.svelte (hunk 5 above)
Comment (valeriegc) [reply src/routes/Canvas.svelte]: it wasnt. removed.

**Review by AlexanderKolberg — APPROVED** [review]
LGTM

**Pull request merged**